tests/todo.spec.js

```diff
@@ -23,6 +23,15 @@ describe('Test GET /todos', () => {
   })
 })
 
```

Comment:
add new test GET /todos?status="DONE" and it should return todo array with some elements

expect(response.length).toEqual(3)